crates/store/src/errors.rs

```diff
@@ -117,105 +111,105 @@ impl From<DatabaseError> for Status {
     GenesisBlockError(#[from] GenesisError),
-    #[error("Pool build error: {0}")]
+    #[error("pool build error")]
     PoolBuildError(#[from] deadpool_sqlite::BuildError),
-    #[error("SQLite migration error: {0}")]
+    #[error("SQLite migration error")]
     SqliteMigrationError(#[from] rusqlite_migration::Error),
 }
 
 #[derive(Debug, Error)]
 pub enum GenesisError {
     // ERRORS WITH AUTOMATIC CONVERSIONS FROM NESTED ERROR TYPES
     // ---------------------------------------------------------------------------------------------
-    #[error("Database error: {0}")]
+    #[error("database error")]
     DatabaseError(#[from] DatabaseError),
-    #[error("Block error: {0}")]
+    #[error("block error")]
     BlockError(#[from] BlockError),
-    #[error("Merkle error: {0}")]
+    #[error("merkle error")]
     MerkleError(#[from] MerkleError),
+    #[error("failed to deserialize genesis file")]
+    GenesisFileDeserializationError(#[from] DeserializationError),
+    #[error("retrieving genesis block header failed")]
+    SelectBlockHeaderByBlockNumError(#[from] Box<DatabaseError>),
 
     // OTHER ERRORS
     // ---------------------------------------------------------------------------------------------
-    #[error("Apply block failed: {0}")]
+    #[error("apply block failed: {0}")]
     ApplyBlockFailed(String),
-    #[error("Failed to read genesis file \"{genesis_filepath}\": {error}")]
+    #[error("failed to read genesis file \"{genesis_filepath}\": {error}")]
     FailedToReadGenesisFile {
         genesis_filepath: String,
         error: io::Error,
     },
-    #[error("Block header in store doesn't match block header in genesis file. Expected {expected_genesis_header:?}, but store contained {block_header_in_store:?}")]
+    #[error("block header in store doesn't match block header in genesis file. Expected {expected_genesis_header:?}, but store contained {block_header_in_store:?}")]
     GenesisBlockHeaderMismatch {
         expected_genesis_header: Box<BlockHeader>,
         block_header_in_store: Box<BlockHeader>,
     },
-    #[error("Failed to deserialize genesis file: {0}")]
-    GenesisFileDeserializationError(DeserializationError),
-    #[error("Retrieving genesis block header failed: {0}")]
-    SelectBlockHeaderByBlockNumError(Box<DatabaseError>),
 }
 
 // ENDPOINT ERRORS
 // =================================================================================================
 #[derive(Error, Debug)]
 pub enum InvalidBlockError {
-    #[error("Duplicated nullifiers {0:?}")]
+    #[error("duplicated nullifiers {0:?}")]
     DuplicatedNullifiers(Vec<Nullifier>),
-    #[error("Invalid output note type: {0:?}")]
+    #[error("invalid output note type: {0:?}")]
     InvalidOutputNoteType(Box<OutputNote>),
-    #[error("Invalid tx hash: expected {expected}, but got {actual}")]
+    #[error("invalid tx hash: expected {expected}, but got {actual}")]
     InvalidTxHash { expected: RpoDigest, actual: RpoDigest },
```

Comment:
```suggestion
    #[error("invalid block tx hash: expected {expected}, but got {actual}")]
    InvalidBlockTxHash { expected: RpoDigest, actual: RpoDigest },
```

Nit: This seems to be specific to the block tx hash.